SyntaxError on `yo keystone`

Hi, was hoping to test out keystone for a site but ran into an issue with running the generator.

* generator-keystone `3.1.6`
* Yo `1.8.1`
* Node `4.4.4`

All works fine with answering the questions but as soon as I answer the `include extra code comments` (answer does not affect outcome), I get the following `SyntaxError`.

```
? ------------------------------------------------
    Finally, would you like to include extra code comments in
    your project? If you're new to Keystone, these may be helpful. No
/usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/node_modules/lodash/dist/lodash.js:6309
        throw e;
        ^

SyntaxError: Unexpected token =
    at Function (native)
    at Function.template (/usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/node_modules/lodash/dist/lodash.js:6306:22)
    at KeystoneGenerator.underscore [as _engine] (/usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/lib/util/engines.js:31:14)
    at KeystoneGenerator.engine (/usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/lib/actions/actions.js:314:10)
    at KeystoneGenerator.template (/usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/lib/actions/actions.js:289:15)
    at KeystoneGenerator.project (/usr/local/lib/node_modules/generator-keystone/app/index.js:289:7)
    at /usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/lib/base.js:395:14
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

I'm only seeing `0.3.15` on npm https://www.npmjs.com/package/generator-keystone

```
npm ERR! No compatible version found: generator-keystone@0.3.17
npm ERR! Valid install targets:
npm ERR! 0.3.15, 0.3.14, 0.3.13, 0.3.12, .....
```

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -31,7 +31,7 @@
     "grunt-concurrent": "^0.5.0",
     "grunt-nodemon": "^0.2.1", <% if (preprocessor === 'sass') { %>
     "grunt-sass": "^1.0.0", <% } else if (preprocessor === 'less' ) { %>
-    "grunt-contrib-less": "^1.3.0", <% } else if (preprocessor ==== 'stylus') { %>
+    "grunt-contrib-less": "^1.3.0", <% } else if (preprocessor === 'stylus') { %>
     "grunt-contrib-stylus": "^1.2.0", <% } %>
     "open": "0.0.5"
   },<% } else if (taskRunner === 'gulp') { %>

